CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ Notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 Kubeclient release versioning follows [SemVer](https://semver.org/).
 
+## 3.1.1 - 2018-05-31
```

Comment:
should say `## Next`

Reply:
OK, took out bump, changed this to "Unreleased"